app/assets/stylesheets/pageflow/themes/legacy/indicators.scss

```diff
@@ -0,0 +1,62 @@
+.entry {
+  .indicator {
+    color: #e6e6e6;
+
+    text-transform: uppercase;
+    text-shadow: 1px 1px 4px rgba(43, 43, 43, 1);
```

Comment:
Color literals like `rgba(43, 43, 43, 1)` should only be used in variable declarations; they should be referred to via variable everywhere else.